pymc/distributions/multivariate.py

```diff
@@ -1707,6 +1719,14 @@ def dist(
 
         return super().dist([mu, rowchol_cov, colchol_cov], **kwargs)
 
+    def get_moment(rv, size, mu, rowchol, colchol):
+        moment = mu
+        if not rv_size_is_none(size):
+            if isinstance(size, int):
+                size = (size,)
+            moment = at.full((*size, mu.size), moment)
+        return moment
```

Comment:
`mu` also needs to be broadcasted to match `rowchol` and `cholchol` dimensions. We can reuse the logic to broadcast `mu` [implemented in `MatrixNormalRV.rng_fn` method](https://github.com/pymc-devs/pymc/blob/main/pymc/distributions/multivariate.py#L1548-L1553).

Reply:
Fixed.